src/axom/primal/operators/detail/winding_number_3d_impl.hpp

```diff
@@ -234,8 +234,9 @@ double nurbs_winding_number(const Point<T, 3>& query,
     {
       field_direction = DiscontinuityAxis::x;
     }
-    else if(exterior_x || exterior_z)
+    else
     {
+      SLIC_ASSERT(exterior_x || exterior_z);
```

Comment:
@jcs15c -- this fixes a bug about a potentially uninitialized `field_direction`.
I think if it gets to this third case, we necessarily have `exterior_x || exterior_z`.

The alternative would be to add a `DiscontinuityAxis::none` enum value (or something similar).